### ISSUE TITLE
Error when upgrading Flutter to version 3

### ISSUE THREAD

**Issue by ambogatyrev** [SimformSolutionsPvtLtd/flutter_showcaseview#215]
![image](https://user-images.githubusercontent.com/79789858/168102083-c4a78868-0011-4046-94ba-968b9d80c2be.png)

**Comment by gitHubAdminUser** [comment]
same here, there are not errors but warnings, please remove the null operator in all SchedulerBinding.

../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/get-4.6.1/lib/get_navigation/src/extension_navigation.dart:468:24: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/scheduler/binding.dart').
      SchedulerBinding.instance!.addPostFrameCallback((_) {
                       ^
../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/measure_size.dart:44:22: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    SchedulerBinding.instance!.addPostFrameCallback(postFrameCallback);
                     ^
../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:127:22: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.addPostFrameCallback((_) => showOverlay());
                     ^
../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:134:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!
                   ^
../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:141:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!
                   ^
../../Developer/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:198:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Developer/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!

**Comment by kjessie00** [comment]
the same error


../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/measure_size.dart:44:22: Warning: Operand of null-aware operation '!' has type 'SchedulerBinding' which excludes null.
 - 'SchedulerBinding' is from 'package:flutter/src/scheduler/binding.dart' ('../../Development/flutter/packages/flutter/lib/src/scheduler/binding.dart').
    SchedulerBinding.instance!.addPostFrameCallback(postFrameCallback);
                     ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:127:22: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
      WidgetsBinding.instance!.addPostFrameCallback((_) => showOverlay());
                     ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:134:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!
                   ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:141:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!
                   ^
../../Development/flutter/.pub-cache/hosted/pub.dartlang.org/showcaseview-1.1.5/lib/src/layout_overlays.dart:198:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
 - 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('../../Development/flutter/packages/flutter/lib/src/widgets/binding.dart').
    WidgetsBinding.instance!

**Comment by P-B1101** [comment]
fixed in #216

**Comment by SteveCruise** [comment]
not fixed, where to run `dart fix --apply`  @P-B1101

**Comment by P-B1101** [comment]
No, I mean I forked the package and fixed it. just need to be merged.
if you want to fix it for yourself (the hard way), you have to clone (or fork) the project and run `dart fix --apply` in the terminal and then use your cloned (forked) project as a library (not the official one).
or go the easy way and wait for an update ;)

**Comment by zakblacki** [comment]
> No, I mean I forked the package and fixed it. just need to be merged. if you want to fix it for yourself (the hard way), you have to clone (or fork) the project and run `dart fix --apply` in the terminal and then use your cloned (forked) project as a library (not the official one). or go the easy way and wait for an update ;)

Thanks man we're waiting
